src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -122,7 +122,7 @@ lemma finrank_euclidean_space_fin {n : ℕ} :
 from `E` to `pi_Lp 2` of the subspaces equipped with the `L2` inner product. -/
 def direct_sum.submodule_is_internal.isometry_L2_of_orthogonal_family
   [decidable_eq ι] {V : ι → submodule 𝕜 E} (hV : direct_sum.submodule_is_internal V)
-  (hV' : orthogonal_family 𝕜 V) :
+  (hV' : @orthogonal_family 𝕜 _ _ _ _ (λ i, V i) _ (λ i, (V i).subtypeₗᵢ)) :
```

Comment:
Is there a convenient way to avoid all the new `@` for `orthogonal_family`?
Or maybe these cases are atypical and no @ is required for most uses?

Reply:
These @ are in order to extract the "old case" of orthogonal family, when the linear map into `E` is just the coercion into `E` of a submodule.  (There are unification problems there, the usual issues with pi types.)

I've been lazy with this refactor and only generalized the lemmas I need for #11255, so these @ show up in all the lemmas I _didn't_ generalize.  I hope to go back later and generalize the rest of the lemmas, at which point I hope the @ will mostly go away!